### PR TITLE
content_soft_delete_migration

### PR DESCRIPTION
**Pull request by MateuszUrbanowicz**

### CODE REVIEW

**Comment by AdrianSkierniewski** [comment]
Right now we only need it for Gzero\Entity\Content

File: phpunit.xml [hunk 1]
```diff
@@ -21,6 +21,9 @@
             <directory suffix=".php">./src</directory>
             <exclude>
                 <directory suffix=".php">./vendor</directory>
+                <directory suffix=".php">./tests</directory>
+                <directory suffix=".php">./src/migrations</directory>
```
Comment (AdrianSkierniewski) [inline phpunit.xml]: I removed those files from code coverage report

File: tests/TestCase.php [hunk 2]
```diff
@@ -34,7 +39,14 @@ public function createApplication()
         require $framework . '/Illuminate/Foundation/start.php';
         require __DIR__ . '/../src/Gzero/Core/helpers.php';
 
-        $this->app['artisan']->call('migrate', ['--path' => '../src/migrations']); // Relative to tests/app/
+        if (!self::$initialSetup) {
+            try {
+                $this->app['artisan']->call('migrate:reset');
```
Comment (AdrianSkierniewski) [inline tests/TestCase.php]: Added this handy block code to cover case when we updated migration that was already installed in test DB

**Comment by PSkierniewski** [comment]
:+1:

**Pull request merged**